extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -311,6 +318,15 @@ public void startPersistenceUnits(HibernateOrmRecorder recorder, BeanContainerBu
         recorder.startAllPersistenceUnits(beanContainer.getValue());
     }
 
+    private String getSqlLoadScript() {
+        // Explicit file or default Hibernate ORM file.
+        return hibernateConfig.sqlLoadScript.orElse("import.sql");
```

Comment:
I would make this method return an `Optional` and take into account the `no-file` case directly. That would be cleaner.